### PR TITLE
fixed major sonarqube issues

### PR DESCRIPTION
**Pull request by samodovdi**

### CODE REVIEW

File: src/Lykke.Common/DateTimeUtils.cs [hunk 1]
```diff
@@ -8,22 +8,9 @@ public static class DateTimeUtils
     {
         public const int SecondsInDay = 86400;
 
+        public static string StandartDateTimeMask => "dd.MM.yy HH:mm:ss";
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Common/DateTimeUtils.cs]: Make it readonly/const

File: src/Lykke.Common/Log/LogToMemory.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@ namespace Common.Log
 {
     public static class LogUtils
     {
-        public static readonly string[] GuiHeader =
+        public static string[] GuiHeader => new[]
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Common/Log/LogToMemory.cs]: Will be good to make it ```readonly IReadOnlyList<string>```, but it's breaking change, of course.

File: src/Lykke.Common/Api/Contract/Responses/ErrorResponse.cs [hunk 3]
```diff
@@ -101,27 +101,27 @@ public string GetSummaryMessage()
...
-            if (ModelErrors != null)
+            if (ModelErrors == null) 
+                return sb.ToString();
+            
+            sb.AppendLine();
+
+            foreach (var error in ModelErrors)
             {
-                sb.AppendLine();
+                if (error.Key == null || error.Value == null)
```
Comment (starkmsu) [inline src/Lykke.Common/Api/Contract/Responses/ErrorResponse.cs]: Need to check for error.Value.Count == 0 beside null value check

File: src/Lykke.Common/TimerPeriod.cs [hunk 4]
```diff
@@ -25,6 +25,7 @@ public abstract class TimerPeriod : IStartable, IStopable, ITimerCommand
         private Task _task;
         private CancellationTokenSource _cancellation;
         private bool _isTelemetryDisabled;
+        private bool disposed;
```
Comment (starkmsu) [inline src/Lykke.Common/TimerPeriod.cs]: Name it _disposed - to avoid confusion with local variable

File: src/Lykke.Common/Utils.cs [hunk 5]
```diff
@@ -702,11 +670,9 @@ public static byte ToByte(this bool src)
             {typeof(long), typeof(long)},
             {typeof(ulong), typeof(ulong)},
             {typeof(double), typeof(double)},
-            {typeof(float), typeof(float)},
-
+            {typeof(float), typeof(float)}
```
Comment (starkmsu) [inline src/Lykke.Common/Utils.cs]: Try to preserve last , in collections init, so this row wouldn't be affected with a new commit when adding new values to collection after this row

File: src/Lykke.Common/AsyncQueue.cs [hunk 6]
```diff
@@ -21,8 +21,6 @@ public Task<T> DequeueAsync()
 
         public Task<T> DequeueAsync(CancellationToken ct)
         {
-            if (ct == null) throw new ArgumentNullException("ct");
```
[the remark this answers is not in the document]
Comment (samodovdi) [reply src/Lykke.Common/AsyncQueue.cs]: sonar says its always false and can be removed

**Pull request merged**